modules/ding_seo/ding_seo.info

```diff
@@ -0,0 +1,91 @@
+name = Ding SEO
+description = Provides various SEO functionality to ding2
+core = 7.x
+package = Ding!
+dependencies[] = ctools
+dependencies[] = ding_base
+dependencies[] = ding_eresource
+dependencies[] = ding_event
+dependencies[] = ding_groups
+dependencies[] = ding_library
+dependencies[] = ding_news
+dependencies[] = ding_page
+dependencies[] = ding_permissions
+dependencies[] = features
+dependencies[] = imagecache_token
+dependencies[] = metatag
+dependencies[] = metatag:metatag_opengraph
+dependencies[] = metatag:metatag_verification
+dependencies[] = schema_article
+dependencies[] = schema_event
+dependencies[] = schema_organization
+dependencies[] = strongarm
+dependencies[] = xmlsitemap_custom
```

Comment:
It's enabled so that administrators can add links manually if they are not being picked up automatically for some reason with our current configuration.